jitbuilder/release/.gitignore

```diff
@@ -52,3 +52,4 @@ union
 worklist
 useIncrement
 useCall
+inliningrecfib
```

Comment:
i suppose it's already not completely in alphabetical order (but it's close!). Would you mind putting this new one in the right spot in the order, please?

Reply:
done! now you can say that the list is completely in alphabetical order :)